test/browser/wrapper.jsx

```diff
@@ -0,0 +1,181 @@
+    injectSelectToEnhanceIntoDOM(scratch, id, name, options, selectedOption)
+
+    window.AccessibleTypeahead.enhanceSelectElement({
+      selectElement: document.querySelector('#' + id)
+    })
+
+    const typeaheadInstances = document.querySelectorAll('.typeahead__wrapper')
+    expect(typeaheadInstances.length).to.equal(1)
+
+    const typeaheadInstance = typeaheadInstances[0]
+    expect(typeaheadInstance.innerHTML).to.contain(`id="${id}"`)
+    expect(typeaheadInstance.innerHTML).to.contain(`name="${name}"`)
+
+    const typeaheadOption = typeaheadInstance.querySelector('.typeahead__option')
+    expect(typeaheadOption.textContent).to.equal(options[selectedOption])
+  })
+
+  it('can enhance a select element no name', () => {
+    const id = false
+    const name = false
+    const options = {
+      fr: 'France',
+      de: 'Germany',
+      gb: 'United Kingdom'
+    }
+    injectSelectToEnhanceIntoDOM(scratch, id, name, options)
+
+    window.AccessibleTypeahead.enhanceSelectElement({
+      selectElement: document.querySelector('select')
+    })
+
+    const typeaheadInstances = document.querySelectorAll('.typeahead__wrapper')
+    expect(typeaheadInstances.length).to.equal(1)
+
+    const typeaheadInstance = typeaheadInstances[0]
+    expect(typeaheadInstance.innerHTML).to.contain(`id=""`)
+    expect(typeaheadInstance.innerHTML).to.contain(`name=""`)
+  })
+
+  it('can overwrite selectedOption with defaultValue', () => {
+    const id = 'location-picker-id'
+    const name = 'location-picker-name'
+    const options = {
+      fr: 'France',
+      de: 'Germany',
+      gb: 'United Kingdom'
+    }
+    const selectedOption = 'gb'
+    injectSelectToEnhanceIntoDOM(scratch, id, name, options, selectedOption)
+
+    window.AccessibleTypeahead.enhanceSelectElement({
+      defaultValue: 'France',
+      selectElement: document.querySelector('#' + id)
+    })
+
+    const typeaheadInstances = document.querySelectorAll('.typeahead__wrapper')
+    const typeaheadInstance = typeaheadInstances[0]
+    const typeaheadOption = typeaheadInstance.querySelector('.typeahead__option')
+    expect(typeaheadOption.textContent).to.equal('France')
+  })
+
+  it('has all options when typing', (done) => {
+    const id = 'location-picker-id'
+    const name = 'location-picker-name'
+    const options = {
+      fr: 'France',
+      de: 'Germany',
+      gb: 'United Kingdom'
+    }
+    injectSelectToEnhanceIntoDOM(scratch, id, name, options)
+
+    window.AccessibleTypeahead.enhanceSelectElement({
+      selectElement: document.querySelector('#' + id)
+    })
+
+    const typeaheadInstances = document.querySelectorAll('.typeahead__wrapper')
+    const typeaheadInstance = typeaheadInstances[0]
+    const typeaheadInput = typeaheadInstance.querySelector('.typeahead__input')
+    const typeaheadOption = typeaheadInstance.querySelector('.typeahead__option')
+
+    // Using setTimeouts here since changes in values take a while to reflect in lists
```

Comment:
Same, this felt v flakey (with too much knowledge of the internal polling) but wanted to get the PR infront of you asap.